text/068-coqide-split.md

```diff
@@ -0,0 +1,102 @@
+---
+Title: Future of CoqIDE
+Drivers: Karl Palmskog (@palmskog), Théo Zimmermann (@Zimmi48)
+---
+
+## Summary
+
+The goal of this CEP is:
+
+1. to clarify the priority and focus shift by the Coq team away from maintaining and evolving CoqIDE;
+2. to give an opportunity to the community of CoqIDE users to assume maintenance  of and lead the future evolution of CoqIDE through a source code split out of the Coq GitHub repository.
+
+More specifically, the proposal is to make an open call for new volunteer maintainers of CoqIDE through Coq community channels, and then to proceed with a source code split only if volunteer maintainers are found.
+
+## Context and motivation
+
+[CoqIDE](https://coq.inria.fr/refman/practical-tools/coqide.html) is an Integrated Development Environment (IDE) for Coq. CoqIDE is implemented using the OCaml programming language and the GTK3 widget toolkit for Graphical User Interfaces (GUIs), thanks to the [lablgtk3](https://opam.ocaml.org/packages/lablgtk3/) OCaml package.
+
+CoqIDE communicates with Coq thanks to an [XML protocol](https://github.com/coq/coq/blob/master/dev/doc/xml-protocol.md), implemented by the coqidetop server (in the coqide-server opam package), which itself is based on a component in Coq called the [State Transaction Machine](https://coq.github.io/doc/master/api/coq-core/Stm/index.html) (STM).
+
```

Comment:
I feel like these are really 3 components (document manager = STM, transport layer = pretty thin XML component). In my experience people seem to get confused about the roles here.